src/userlist/UsersCard.js

```diff
@@ -32,6 +34,10 @@ export default class UsersCard extends Component {
     filter: string,
   };
 
+  static contextTypes = {
+    drawer: () => null,
```

Comment:
It is either going to be part of the Navigation state, which is WIP or just a temporary component state.

Any state that does not need to be accessed from anywhere else, is recommended (officially) to not be in Redux, but be in the component state.